### ISSUE TITLE
About the result of baseline methods in table II

### ISSUE THREAD

**Issue by yxxxxx38324** [MCZhi/Predictive-Decision#2]
Hi, thanks for your work!

I have some questions about the success rate about the baseline method in table II.
I have used the pretrained model provided in the repo for the test.py, but the success rate of "Ours" methods is only 80% for the first scene, which is lower than the 98% in the paper.

My command line is:
nohup python test.py --name baseline --use_interaction --model_path PATH/interaction_aware_predictor.pth 

Did I make any mistakes in my reproduction steps?

**Comment by MCZhi** [comment]
Hi @yxxxxx38324,

Thank you for your interest in our work! 

The discrepancy you observed in the success rate might be due to the use of different environments between the code and our experiments. In the code, we provide the environment given by the SMARTS competition for convenience. However, in our experiments, we manually build the scenarios and might use a different random seed. So may want to try training a model on the env provided in the code, which may bring better performance. Please let us know if you have any further questions or concerns.

**Comment by yxxxxx38324** [comment]
> Hi @yxxxxx38324,
> 
> Thank you for your interest in our work!
> 
> The discrepancy you observed in the success rate might be due to the use of different environments between the code and our experiments. In the code, we provide the environment given by the SMARTS competition for convenience. However, in our experiments, we manually build the scenarios and might use a different random seed. So may want to try training a model on the env provided in the code, which may bring better performance. Please let us know if you have any further questions or concerns.

Thank you for your reply. 
I did try to retrain using the environment provided in the code, and the result was 0.8827. Moreover, after removing interaction, the result is 0.9444. Maybe there is a problem with my baseline training, and I will try again to determine what the problem is.

**Comment by yxxxxx38324** [comment]
Could you please provide weights for using the environment training in the code? What other ways can I reproduce the results of the experiment, such as use another random seed? Where can I set the random seed?

It seems that there were no obvious problems with the way I trained, but I still can't get the results in the paper. 
Thanks a lot!

**Comment by MCZhi** [comment]
Hi, @yxxxxx38324, sorry for the late reply.

Unfortunately, I have no time to train the network using the provided environment. However, you can directly create a HiWayEnv using SMARTS, build the scenarios provided by the competition, and set the random seed in the environment. This should help you ensure reproducibility.

**Comment by yxxxxx38324** [comment]
> Hi, @yxxxxx38324, sorry for the late reply.
> 
> Unfortunately, I have no time to train the network using the provided environment. However, you can directly create a HiWayEnv using SMARTS, build the scenarios provided by the competition, and set the random seed in the environment. This should help you ensure reproducibility.

Thanks for your advice!
